src/data/ingredientToIngredientAliases.ts

```diff
@@ -155,6 +155,21 @@ const ingredientToIngredientAliases: IngredientToIngredientAliases = {
   "Boiling Water": [],
   "Cinnamon": [],
   "Pumpkin Syrup": [],
+  "Brandy": [],
+  "Apple Cider": [],
+  "Pear Brandy": [],
+  "Hard Apple Cider": [],
+  "Dry Alsatian Riesling": [],
+  "Star Anise Pod": [],
+  "Ginger": [],
+  "Whole Cloves": [],
+  "Cardamom Pod": [],
```

Comment:
I think (probably) that if you search "Cardamom", recipes requiring this ingredient should show up too, no? If so, we should add that alias.